docs/component-guides/profiles.md

```diff
@@ -1,68 +1,72 @@
+The following components have Profile-level support: 
+- Central Dashboard
+- Notebooks
+- Pipelines
+- AutoML (Katib)
+- KFServing
 
-The `default-editor` SA is used by various services in Kubeflow to launch resources in users namespace and hence the pods inherit the permissions of this service account. However, not all services do this by default.
+The following components have been tested to work with the `AwsIamForServiceAccount` plugin: 
+- Notebooks
 
-#### Component Level Implementations
+Integration with the `AwsIamForServiceAccount` plugin is being actively worked on for all components with Profile-level support. 
 
-As of writing, the following components have profile level support: Central Dashboard, Notebooks, Pipelines, AutoML (Katib), KFServing.
+You can find documentation about the `AwsIamForServiceAccount` plugin for specific components in the individual [component guides](/docs/component-guides/).
 
-The following components have been tested to work with the `AwsIamForServiceAccount` plugin: Notebooks.
+## Configuration steps
 
-Support for the `AwsIamForServiceAccount` plugin in all the components that have profile level support is being actively worked on.
+After installing Kubeflow on AWS with one of the available [deployment options](/docs/deployment/install/), you can configure Kubeflow Profiles with the following steps: 
 
-Documentation and samples on using the `AwsIamForServiceAccount` plugin in the [component guides](./) for the respective component.
-
-### Configuration Steps
-
-The following steps should be run after deploying Kubeflow via the [provided deployment options](../../docs/deployment).
-
-1. Export the following variables for convenience.
-    ```
+1. Define the following environment variables:
+    ```bash
     export CLUSTER_NAME=<your cluster name>
     export CLUSTER_REGION=<your region>
     export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
     export PROFILE_NAME=<the name of the profile to be created>
     ```
 
-1. Create an IAM policy using [the following policy document](../../awsconfigs/infra_configs/iam_profile_controller_policy.json).
-    ```
+2. Create an IAM policy using the [IAM Profile controller policy](https://github.com/awslabs/kubeflow-manifests/blob/main/awsconfigs/infra_configs/iam_profile_controller_policy.json) file.
```

Comment:
relative link?

Reply:
Any code sample files won't be a docs page so we can't use the Hugo relative link for those.